etcdmain/help.go

```diff
@@ -116,6 +116,8 @@ security flags:
 		enable peer client cert authentication.
 	--peer-trusted-ca-file ''
 		path to the peer server TLS trusted CA file.
+	--peer-auto-tls 'false'
+		peer TLS using generated certificates.
```

Comment:
peer TLS using generated certificates if --peer-key-file and --peer-cert-file are not provided.